build-tools/automation/azure-pipelines.yaml

```diff
@@ -55,6 +55,8 @@ variables:
   value: $[or(startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), startsWith(variables['System.PullRequest.TargetBranch'], 'release/'))]
 - name: DotNetNUnitCategories
   value: '& TestCategory != DotNetIgnore & TestCategory != HybridAOT & TestCategory != MkBundle & TestCategory != MonoSymbolicate & TestCategory != PackagesConfig & TestCategory != StaticProject & TestCategory != SystemApplication'
+- name: ExcludedNUnitCategories
+  value: '& cat != DotNetIgnore & cat != HybridAOT & cat != MkBundle & cat != MonoSymbolicate & cat != PackagesConfig & cat != StaticProject & cat != SystemApplication'
```

Comment:
This seems duplicate of `DotNetNUnitCategories`, why is the syntax here different?

Reply:
`dotnet test` uses one syntax, `NUnit.Engine` uses a different syntax.

When you use `dotnet test` the adapter does the translation.  Because `dotnet-test-slicer` hits `NUnit.Engine` directly, we need to use NUnit's [test selection language](https://docs.nunit.org/articles/nunit/running-tests/Test-Selection-Language.html).

I put a little note here https://github.com/xamarin/xamarin-android/blob/f0a24bf67b3f0e56ed7a1d24c2e3789f542f0908/build-tools/automation/yaml-templates/run-sliced-nunit-tests.yaml#L3, but it doesn't bubble up to that variable specification.